dashboard/pkg/epinio/edit/services.vue

```diff
@@ -51,7 +55,7 @@ export default Vue.extend<Data, EpinioCompRecord, EpinioCompRecord, EpinioCompRe
       this.mixinFetch()
     ]);
 
-    Vue.set(this.value, 'catalog_service', this.$route.query[EPINIO_SERVICE_PARAM]);
+    Vue.set(this.value, 'catalog_service', this.selectedCatalogService?.meta.name || null);
```

Comment:
this breaks autopopulating the service when clicking on a service from the home page

Reply:
Fixed